is this normal?

the doc build seems to have a large set of unexpected dependencies...
```
mike@mike-VirtualBox:~/go/src/github.com/openSUSE/umoci$ sudo make install
docker build -t umoci_dev --build-arg DOCKER_IMAGE=opensuse/amd64:tumbleweed .
Sending build context to Docker daemon  52.89MB
Step 1/13 : FROM opensuse/leap:latest
latest: Pulling from opensuse/leap
81e5221d78cb: Pull complete 
Digest: sha256:50aaf6277e48469b312cdf6d98aa220219809e99079d090a2613a055f3a969f0
Status: Downloaded newer image for opensuse/leap:latest
 ---> 3ee512152fcf
Step 2/13 : MAINTAINER "Aleksa Sarai <asarai@suse.com>"
 ---> Running in 429941c74034
Removing intermediate container 429941c74034
 ---> f3712cf94aa6
Step 3/13 : RUN zypper ar -f -p 10 -g obs://Virtualization:containers obs-vc && 	zypper ar -f -p 10 -g obs://home:cyphar:bats obs-bats && 	zypper --gpg-auto-import-keys -n ref && 	zypper -n up
 ---> Running in 2b3895a58c94
Guessed: platform = openSUSE_Leap_$releasever
Adding repository 'obs-vc' [......done]
Repository 'obs-vc' successfully added

URI         : https://download.opensuse.org/repositories/Virtualization:/containers/openSUSE_Leap_15.1
Enabled     : Yes                                                                                     
GPG Check   : Yes                                                                                     
Autorefresh : Yes                                                                                     
Priority    : 10 (raised priority)                                                                    

Repository priorities in effect: (See 'zypper lr -P' for details)
      10 (raised priority)  :  1 repository  
      99 (default priority) :  4 repositories
Guessed: platform = openSUSE_Leap_$releasever
Adding repository 'obs-bats' [......done]
Repository 'obs-bats' successfully added

URI         : https://download.opensuse.org/repositories/home:/cyphar:/bats/openSUSE_Leap_15.1
Enabled     : Yes                                                                             
GPG Check   : Yes                                                                             
Autorefresh : Yes                                                                             
Priority    : 10 (raised priority)                                                            

Repository priorities in effect: (See 'zypper lr -P' for details)
      10 (raised priority)  :  2 repositories
      99 (default priority) :  4 repositories
Retrieving repository 'obs-bats' metadata [...

Automatically importing the following key:

  Repository:       obs-bats                                                
  Key Name:         home:cyphar OBS Project <home:cyphar@build.opensuse.org>
  Key Fingerprint:  A8728192 039A8CD6 EEF59A34 102C55BA 3E2DB1F2            
  Key Created:      Thu Feb 22 06:42:32 2018                                
  Key Expires:      Sat May  2 06:42:32 2020 (EXPIRED)                      
  Rpm Name:         gpg-pubkey-3e2db1f2-5a8e6658                            


Warning: The gpg key signing file 'repomd.xml' has expired.
  Repository:       obs-bats                                                
  Key Name:         home:cyphar OBS Project <home:cyphar@build.opensuse.org>
  Key Fingerprint:  A8728192 039A8CD6 EEF59A34 102C55BA 3E2DB1F2            
  Key Created:      Thu Feb 22 06:42:32 2018                                
  Key Expires:      Sat May  2 06:42:32 2020 (EXPIRED)                      
  Rpm Name:         gpg-pubkey-3e2db1f2-5a8e6658                            
done]
Building repository 'obs-bats' cache [....done]
Retrieving repository 'obs-vc' metadata [...

Automatically importing the following key:

  Repository:       obs-vc                                                        
  Key Name:         Virtualization OBS Project <Virtualization@build.opensuse.org>
  Key Fingerprint:  55A0B34D 49501BB7 CA474F5A A193FBB5 72174FC2                  
  Key Created:      Mon Feb 17 05:10:11 2020                                      
  Key Expires:      Wed Apr 27 05:10:11 2022                                      
  Rpm Name:         gpg-pubkey-72174fc2-5e4a2033                                  


..done]
Building repository 'obs-vc' cache [....done]
Retrieving repository 'Non-OSS Repository' metadata [...done]
Building repository 'Non-OSS Repository' cache [....done]
Retrieving repository 'Main Repository' metadata [............done]
Building repository 'Main Repository' cache [....done]
Retrieving repository 'Main Update Repository' metadata [.....................done]
Building repository 'Main Update Repository' cache [....done]
Retrieving repository 'Update Repository (Non-Oss)' metadata [...done]
Building repository 'Update Repository (Non-Oss)' cache [....done]
All repositories have been refreshed.
Loading repository data...
Reading installed packages...

Nothing to do.
Removing intermediate container 2b3895a58c94
 ---> 53f0fe78fa8c
Step 4/13 : RUN zypper -n in 		bats 		git 		"go>=1.11" 		go-mtree 		jq 		libcap-progs 		make 		moreutils 	oci-image-tools 		oci-runtime-tools 		python-setuptools python-xattr attr 		skopeo 		tar
 ---> Running in eb4ec335d7ae
Loading repository data...
Reading installed packages...
'python-xattr' not found in package names. Trying capabilities.
Resolving package dependencies...

The following 101 NEW packages are going to be installed:
  attr bats binutils blog cpp cpp7 dbus-1 file gcc gcc7 gio-branding-openSUSE git git-core glib-networking glib2-tools glibc-devel gnu_parallel go go-mtree go1.11 gsettings-desktop-schemas jq kbd kbd-legacy kmod less libapparmor1 libarchive13 libargon2-1 libasan4 libatomic1 libcap-progs libcilkrts5 libcontainers-common libcryptsetup12 libdb-4_8 libdbus-1-3 libdevmapper1_03 libedit0 libexpat1 libgdbm4 libgio-2_0-0 libglib-2_0-0 libgmodule-2_0-0 libgobject-2_0-0 libgomp1 libisl15 libitm1 libjq1 libjson-c3 libkmod2 liblsan0 libmpc3 libmpfr6 libmpx2 libmpxwrappers2 libonig4 libostree-1-1 libpcre2-8-0 libpython2_7-1_0 libpython3_6m1_0 libqrencode4 libseccomp2 libsha1detectcoll1 libsoup-2_4-1 libtsan0 libubsan0 linux-glibc-devel make moreutils oci-image-tools oci-runtime-tools openslp openssh pam-config perl perl-Error perl-IO-Tty perl-IPC-Run perl-Time-Duration perl-TimeDate pkg-config python-base python-rpm-macros python-setuptools python-six python-xml python2-cffi python2-pycparser python2-xattr python3 python3-base rsync shared-mime-info skopeo systemd systemd-presets-branding-openSUSE systemd-presets-common-SUSE sysvinit-tools tar udev

The following 3 packages require a system reboot:
  dbus-1 systemd udev

101 new packages to install.
Overall download size: 196.9 MiB. Already cached: 0 B. After the operation, additional 795.7 MiB will be used.

    Note: System reboot required.
Continue? [y/n/v/...? shows all options] (y): y
Retrieving package go-mtree-0.4.4-lp151.1.17.x86_64 (1/101),   1.7 MiB (  4.2 MiB unpacked)
Retrieving: go-mtree-0.4.4-lp151.1.17.x86_64.rpm [..done (960.5 KiB/s)]
Retrieving package libcontainers-common-20200603-lp151.2.1.noarch (2/101),  96.0 KiB ( 89.1 KiB unpacked)
Retrieving: libcontainers-common-20200603-lp151.2.1.noarch.rpm [.done]
Retrieving package oci-image-tools-0.3.0-lp151.1.17.x86_64 (3/101),   5.0 MiB ( 17.1 MiB unpacked)
Retrieving: oci-image-tools-0.3.0-lp151.1.17.x86_64.rpm [....done (1.4 MiB/s)]
Retrieving package oci-runtime-tools-0.2.0+r40-lp151.1.17.x86_64 (4/101),   4.9 MiB ( 16.8 MiB unpacked)
Retrieving: oci-runtime-tools-0.2.0+r40-lp151.1.17.x86_64.rpm [....done (1.3 MiB/s)]
Retrieving package attr-2.4.47-lp151.3.70.x86_64 (5/101),  51.3 KiB (122.0 KiB unpacked)
Retrieving: attr-2.4.47-lp151.3.70.x86_64.rpm [done]
Retrieving package blog-2.18-lp151.5.3.x86_64 (6/101),  53.6 KiB (129.0 KiB unpacked)
Retrieving: blog-2.18-lp151.5.3.x86_64.rpm [.done (17.7 KiB/s)]
Retrieving package gnu_parallel-20180422-lp151.2.1.noarch (7/101), 282.4 KiB (650.8 KiB unpacked)
Retrieving: gnu_parallel-20180422-lp151.2.1.noarch.rpm [done]
Retrieving package gsettings-desktop-schemas-3.24.1-lp151.3.2.x86_64 (8/101),  43.9 KiB (148.9 KiB unpacked)
Retrieving: gsettings-desktop-schemas-3.24.1-lp151.3.2.x86_64.rpm [done]
Retrieving package kbd-legacy-2.0.4-lp151.8.1.noarch (9/101), 502.8 KiB (517.0 KiB unpacked)
Retrieving: kbd-legacy-2.0.4-lp151.8.1.noarch.rpm [.done (15.6 KiB/s)]
Retrieving package kmod-25-lp151.6.1.x86_64 (10/101), 142.7 KiB (289.6 KiB unpacked)
Retrieving: kmod-25-lp151.6.1.x86_64.rpm [done]
Retrieving package libargon2-1-0.0+git20171227.670229c-lp151.3.3.x86_64 (11/101),  19.9 KiB ( 30.2 KiB unpacked)
Retrieving: libargon2-1-0.0+git20171227.670229c-lp151.3.3.x86_64.rpm [done]
Retrieving package libcap-progs-2.25-lp151.3.70.x86_64 (12/101),  24.3 KiB ( 52.3 KiB unpacked)
Retrieving: libcap-progs-2.25-lp151.3.70.x86_64.rpm [.done (15.6 KiB/s)]
Retrieving package libedit0-3.1.snap20150325-lp151.3.3.x86_64 (13/101),  94.3 KiB (222.2 KiB unpacked)
Retrieving: libedit0-3.1.snap20150325-lp151.3.3.x86_64.rpm [done]
Retrieving package libgdbm4-1.12-lp151.2.67.x86_64 (14/101),  75.7 KiB (251.7 KiB unpacked)
Retrieving: libgdbm4-1.12-lp151.2.67.x86_64.rpm [done]
Retrieving package libisl15-0.18-lp151.2.70.x86_64 (15/101), 546.4 KiB (  1.7 MiB unpacked)
Retrieving: libisl15-0.18-lp151.2.70.x86_64.rpm [.done (1.6 MiB/s)]
Retrieving package libjson-c3-0.13-lp151.2.3.x86_64 (16/101),  40.7 KiB ( 64.9 KiB unpacked)
Retrieving: libjson-c3-0.13-lp151.2.3.x86_64.rpm [done]
Retrieving package libkmod2-25-lp151.6.1.x86_64 (17/101),  99.1 KiB (212.5 KiB unpacked)
Retrieving: libkmod2-25-lp151.6.1.x86_64.rpm [done]
Retrieving package libmpfr6-4.0.1-lp151.2.70.x86_64 (18/101), 214.3 KiB (500.6 KiB unpacked)
Retrieving: libmpfr6-4.0.1-lp151.2.70.x86_64.rpm [.done (1.8 MiB/s)]
Retrieving package libmpx2-8.2.1+r264010-lp151.1.33.x86_64 (19/101),  19.1 KiB ( 18.3 KiB unpacked)
Retrieving: libmpx2-8.2.1+r264010-lp151.1.33.x86_64.rpm [done]
Retrieving package libmpxwrappers2-8.2.1+r264010-lp151.1.33.x86_64 (20/101),  17.8 KiB ( 14.2 KiB unpacked)
Retrieving: libmpxwrappers2-8.2.1+r264010-lp151.1.33.x86_64.rpm [done]
Retrieving package libonig4-6.7.0-lp151.2.2.x86_64 (21/101), 126.9 KiB (494.1 KiB unpacked)
Retrieving: libonig4-6.7.0-lp151.2.2.x86_64.rpm [done]
Retrieving package libpcre2-8-0-10.31-lp151.2.3.x86_64 (22/101), 235.5 KiB (648.1 KiB unpacked)
Retrieving: libpcre2-8-0-10.31-lp151.2.3.x86_64.rpm [.done (863.3 KiB/s)]
Retrieving package libqrencode4-4.0.0-lp151.2.3.x86_64 (23/101),  28.8 KiB ( 50.0 KiB unpacked)
Retrieving: libqrencode4-4.0.0-lp151.2.3.x86_64.rpm [done]
Retrieving package libsha1detectcoll1-1.0.3-lp151.3.3.x86_64 (24/101),  23.1 KiB ( 45.8 KiB unpacked)
Retrieving: libsha1detectcoll1-1.0.3-lp151.3.3.x86_64.rpm [done]
Retrieving package linux-glibc-devel-4.15-lp151.2.72.noarch (25/101),   1.3 MiB (  5.7 MiB unpacked)
Retrieving: linux-glibc-devel-4.15-lp151.2.72.noarch.rpm [.done (1.4 MiB/s)]
Retrieving package make-4.2.1-lp151.7.39.x86_64 (26/101), 275.5 KiB (394.6 KiB unpacked)
Retrieving: make-4.2.1-lp151.7.39.x86_64.rpm [.done (468.4 KiB/s)]
Retrieving package pam-config-0.96-lp151.1.2.x86_64 (27/101), 129.0 KiB (626.8 KiB unpacked)
Retrieving: pam-config-0.96-lp151.1.2.x86_64.rpm [done]
Retrieving package pkg-config-0.29.2-lp151.2.70.x86_64 (28/101), 230.1 KiB (675.4 KiB unpacked)
Retrieving: pkg-config-0.29.2-lp151.2.70.x86_64.rpm [.done (2.0 MiB/s)]
Retrieving package systemd-presets-common-SUSE-15-lp151.7.1.noarch (29/101),  19.9 KiB (  4.0 KiB unpacked)
Retrieving: systemd-presets-common-SUSE-15-lp151.7.1.noarch.rpm [done]
Retrieving package tar-1.30-lp151.2.1.x86_64 (30/101), 209.1 KiB (470.6 KiB unpacked)
Retrieving: tar-1.30-lp151.2.1.x86_64.rpm [done]
Retrieving package sysvinit-tools-2.88+-lp151.2.3.x86_64 (31/101), 144.5 KiB (391.3 KiB unpacked)
Retrieving: sysvinit-tools-2.88+-lp151.2.3.x86_64.rpm [.done (1.3 MiB/s)]
Retrieving package kbd-2.0.4-lp151.8.1.x86_64 (32/101),   1.8 MiB (  4.2 MiB unpacked)
Retrieving: kbd-2.0.4-lp151.8.1.x86_64.rpm [.done (1.6 MiB/s)]
Retrieving package libmpc3-1.1.0-lp151.2.70.x86_64 (33/101),  48.3 KiB ( 99.5 KiB unpacked)
Retrieving: libmpc3-1.1.0-lp151.2.70.x86_64.rpm [done]
Retrieving package systemd-presets-branding-openSUSE-12.2-lp151.13.2.noarch (34/101),  17.6 KiB (  184   B unpacked)
Retrieving: systemd-presets-branding-openSUSE-12.2-lp151.13.2.noarch.rpm [.done (236.2 KiB/s)]
Retrieving package binutils-2.32-lp151.3.6.1.x86_64 (35/101),   6.0 MiB ( 47.6 MiB unpacked)
Retrieving: binutils-2.32-lp151.3.6.1.x86_64.rpm [...done (2.5 MiB/s)]
Retrieving package file-5.32-lp151.8.3.1.x86_64 (36/101),  50.4 KiB ( 81.3 KiB unpacked)
Retrieving: file-5.32-lp151.8.3.1.x86_64.rpm [done]
Retrieving package libapparmor1-2.12.3-lp151.4.3.1.x86_64 (37/101),  35.0 KiB ( 67.5 KiB unpacked)
Retrieving: libapparmor1-2.12.3-lp151.4.3.1.x86_64.rpm [done]
Retrieving package libarchive13-3.3.2-lp151.5.3.1.x86_64 (38/101), 302.1 KiB (755.7 KiB unpacked)
Retrieving: libarchive13-3.3.2-lp151.5.3.1.x86_64.rpm [.done (15.6 KiB/s)]
Retrieving package libasan4-7.5.0+r278197-lp151.2.12.1.x86_64 (39/101), 358.2 KiB (  1.3 MiB unpacked)
Retrieving: libasan4-7.5.0+r278197-lp151.2.12.1.x86_64.rpm [done]
Retrieving package libatomic1-9.3.1+git1296-lp151.2.2.x86_64 (40/101),  25.1 KiB ( 26.1 KiB unpacked)
Retrieving: libatomic1-9.3.1+git1296-lp151.2.2.x86_64.rpm [done]
Retrieving package libcilkrts5-7.5.0+r278197-lp151.2.12.1.x86_64 (41/101),  61.4 KiB (120.1 KiB unpacked)
Retrieving: libcilkrts5-7.5.0+r278197-lp151.2.12.1.x86_64.rpm [.done (15.6 KiB/s)]
Retrieving package libdb-4_8-4.8.30-lp151.7.3.1.x86_64 (42/101), 693.5 KiB (  3.1 MiB unpacked)
Retrieving: libdb-4_8-4.8.30-lp151.7.3.1.x86_64.rpm [done]
Retrieving package libdbus-1-3-1.12.2-lp151.4.3.1.x86_64 (43/101), 150.7 KiB (325.9 KiB unpacked)
Retrieving: libdbus-1-3-1.12.2-lp151.4.3.1.x86_64.rpm [done]
Retrieving package libdevmapper1_03-1.02.149-lp151.4.12.1.x86_64 (44/101), 167.6 KiB (345.8 KiB unpacked)
Retrieving: libdevmapper1_03-1.02.149-lp151.4.12.1.x86_64.rpm [.done (1.0 MiB/s)]
Retrieving package libexpat1-2.2.5-lp151.3.6.1.x86_64 (45/101),  75.0 KiB (198.3 KiB unpacked)
Retrieving: libexpat1-2.2.5-lp151.3.6.1.x86_64.rpm [done]
Retrieving package libglib-2_0-0-2.54.3-lp151.7.3.1.x86_64 (46/101), 519.2 KiB (  1.5 MiB unpacked)
Retrieving: libglib-2_0-0-2.54.3-lp151.7.3.1.x86_64.rpm [done]
Retrieving package libgomp1-9.3.1+git1296-lp151.2.2.x86_64 (47/101), 123.4 KiB (288.1 KiB unpacked)
Retrieving: libgomp1-9.3.1+git1296-lp151.2.2.x86_64.rpm [done]
Retrieving package libitm1-9.3.1+git1296-lp151.2.2.x86_64 (48/101),  43.1 KiB (110.4 KiB unpacked)
Retrieving: libitm1-9.3.1+git1296-lp151.2.2.x86_64.rpm [.done (15.6 KiB/s)]
Retrieving package liblsan0-9.3.1+git1296-lp151.2.2.x86_64 (49/101), 150.1 KiB (349.2 KiB unpacked)
Retrieving: liblsan0-9.3.1+git1296-lp151.2.2.x86_64.rpm [done]
Retrieving package libpython2_7-1_0-2.7.17-lp151.10.17.1.x86_64 (50/101), 832.4 KiB (  2.2 MiB unpacked)
Retrieving: libpython2_7-1_0-2.7.17-lp151.10.17.1.x86_64.rpm [done]
Retrieving package libpython3_6m1_0-3.6.10-lp151.6.18.1.x86_64 (51/101),   1.1 MiB (  3.2 MiB unpacked)
Retrieving: libpython3_6m1_0-3.6.10-lp151.6.18.1.x86_64.rpm [.done (15.6 KiB/s)]
Retrieving package libseccomp2-2.4.1-lp151.3.3.1.x86_64 (52/101),  57.1 KiB (306.2 KiB unpacked)
Retrieving: libseccomp2-2.4.1-lp151.3.3.1.x86_64.rpm [done]
Retrieving package libtsan0-9.3.1+git1296-lp151.2.2.x86_64 (53/101), 301.4 KiB (965.2 KiB unpacked)
Retrieving: libtsan0-9.3.1+git1296-lp151.2.2.x86_64.rpm [.done (15.6 KiB/s)]
Retrieving package libubsan0-7.5.0+r278197-lp151.2.12.1.x86_64 (54/101), 134.6 KiB (305.3 KiB unpacked)
Retrieving: libubsan0-7.5.0+r278197-lp151.2.12.1.x86_64.rpm [done]
Retrieving package openslp-2.0.0-lp151.7.9.1.x86_64 (55/101),  65.0 KiB (139.1 KiB unpacked)
Retrieving: openslp-2.0.0-lp151.7.9.1.x86_64.rpm [done]
Retrieving package python-rpm-macros-20200207.5feb6c1-lp151.2.6.1.noarch (56/101),  27.9 KiB ( 50.6 KiB unpacked)
Retrieving: python-rpm-macros-20200207.5feb6c1-lp151.2.6.1.noarch.rpm [done]
Retrieving package libjq1-1.6-lp151.3.3.1.x86_64 (57/101), 118.2 KiB (304.3 KiB unpacked)
Retrieving: libjq1-1.6-lp151.3.3.1.x86_64.rpm [.done (15.6 KiB/s)]
Retrieving package glibc-devel-2.26-lp151.19.15.1.x86_64 (58/101), 755.5 KiB (  2.9 MiB unpacked)
Retrieving: glibc-devel-2.26-lp151.19.15.1.x86_64.rpm [done]
Retrieving package cpp7-7.5.0+r278197-lp151.2.12.1.x86_64 (59/101),   7.9 MiB ( 22.1 MiB unpacked)
Retrieving: cpp7-7.5.0+r278197-lp151.2.12.1.x86_64.rpm [....done (2.6 MiB/s)]
Retrieving package perl-5.26.1-lp151.9.3.1.x86_64 (60/101),   6.5 MiB ( 40.3 MiB unpacked)
Retrieving: perl-5.26.1-lp151.9.3.1.x86_64.rpm [...done (2.3 MiB/s)]
Retrieving package dbus-1-1.12.2-lp151.4.3.1.x86_64 (61/101), 249.6 KiB (624.4 KiB unpacked)
Retrieving: dbus-1-1.12.2-lp151.4.3.1.x86_64.rpm [.done (871.3 KiB/s)]
Retrieving package libgobject-2_0-0-2.54.3-lp151.7.3.1.x86_64 (62/101), 161.3 KiB (336.8 KiB unpacked)
Retrieving: libgobject-2_0-0-2.54.3-lp151.7.3.1.x86_64.rpm [done]
Retrieving package libgmodule-2_0-0-2.54.3-lp151.7.3.1.x86_64 (63/101),  50.6 KiB ( 14.3 KiB unpacked)
Retrieving: libgmodule-2_0-0-2.54.3-lp151.7.3.1.x86_64.rpm [done]
Retrieving package rsync-3.1.3-lp151.4.3.1.x86_64 (64/101), 305.7 KiB (575.7 KiB unpacked)
Retrieving: rsync-3.1.3-lp151.4.3.1.x86_64.rpm [.done (2.2 MiB/s)]
Retrieving package python3-base-3.6.10-lp151.6.18.1.x86_64 (65/101),   7.7 MiB ( 31.1 MiB unpacked)
Retrieving: python3-base-3.6.10-lp151.6.18.1.x86_64.rpm [.....done (1.8 MiB/s)]
Retrieving package python-base-2.7.17-lp151.10.17.1.x86_64 (66/101),   5.9 MiB ( 23.6 MiB unpacked)
Retrieving: python-base-2.7.17-lp151.10.17.1.x86_64.rpm [..done (3.0 MiB/s)]
Retrieving package jq-1.6-lp151.3.3.1.x86_64 (67/101),  65.7 KiB (101.9 KiB unpacked)
Retrieving: jq-1.6-lp151.3.3.1.x86_64.rpm [.done (16.3 KiB/s)]
Retrieving package gcc7-7.5.0+r278197-lp151.2.12.1.x86_64 (68/101),  19.7 MiB ( 72.7 MiB unpacked)
Retrieving: gcc7-7.5.0+r278197-lp151.2.12.1.x86_64.rpm [........done (2.4 MiB/s)]
Retrieving package perl-TimeDate-2.30-lp151.3.6.1.noarch (69/101),  49.3 KiB ( 93.6 KiB unpacked)
Retrieving: perl-TimeDate-2.30-lp151.3.6.1.noarch.rpm [.done (17.6 KiB/s)]
Retrieving package python3-3.6.10-lp151.6.18.1.x86_64 (70/101),  66.8 KiB (125.9 KiB unpacked)
Retrieving: python3-3.6.10-lp151.6.18.1.x86_64.rpm [done]
Retrieving package python-xml-2.7.17-lp151.10.17.1.x86_64 (71/101), 208.8 KiB (910.9 KiB unpacked)
Retrieving: python-xml-2.7.17-lp151.10.17.1.x86_64.rpm [done]
Retrieving package bats-1.1.0+cyphar-lp151.21.1.noarch (72/101),  27.0 KiB ( 43.9 KiB unpacked)
Retrieving: bats-1.1.0+cyphar-lp151.21.1.noarch.rpm [.done]
Retrieving package less-530-lp151.2.38.x86_64 (73/101), 134.0 KiB (293.9 KiB unpacked)
Retrieving: less-530-lp151.2.38.x86_64.rpm [.done]
Retrieving package libcryptsetup12-2.0.5-lp151.1.17.x86_64 (74/101), 157.4 KiB (329.3 KiB unpacked)
Retrieving: libcryptsetup12-2.0.5-lp151.1.17.x86_64.rpm [done]
Retrieving package shared-mime-info-1.9-lp151.3.20.x86_64 (75/101), 416.0 KiB (  5.1 MiB unpacked)
Retrieving: shared-mime-info-1.9-lp151.3.20.x86_64.rpm [.done (12.0 KiB/s)]
Retrieving package cpp-7-lp151.3.5.x86_64 (76/101),   9.1 KiB (   82   B unpacked)
Retrieving: cpp-7-lp151.3.5.x86_64.rpm [done]
Retrieving package perl-Time-Duration-1.200000-lp151.2.1.noarch (77/101),  25.7 KiB ( 38.2 KiB unpacked)
Retrieving: perl-Time-Duration-1.200000-lp151.2.1.noarch.rpm [.done (221.2 KiB/s)]
Retrieving package perl-IO-Tty-1.12-lp151.2.2.x86_64 (78/101),  44.2 KiB ( 81.1 KiB unpacked)
Retrieving: perl-IO-Tty-1.12-lp151.2.2.x86_64.rpm [done]
Retrieving package perl-Error-0.17025-lp151.2.1.noarch (79/101),  40.5 KiB ( 70.5 KiB unpacked)
Retrieving: perl-Error-0.17025-lp151.2.1.noarch.rpm [done]
Retrieving package python2-pycparser-2.17-lp151.2.1.noarch (80/101), 185.1 KiB (  1.0 MiB unpacked)
Retrieving: python2-pycparser-2.17-lp151.2.1.noarch.rpm [done]
Retrieving package gio-branding-openSUSE-42.1-lp151.4.2.noarch (81/101),  24.9 KiB (  8.1 KiB unpacked)
Retrieving: gio-branding-openSUSE-42.1-lp151.4.2.noarch.rpm [.done]
Retrieving package gcc-7-lp151.3.5.x86_64 (82/101),  10.1 KiB (    0   B unpacked)
Retrieving: gcc-7-lp151.3.5.x86_64.rpm [done]
Retrieving package perl-IPC-Run-0.99-lp151.2.1.noarch (83/101), 138.1 KiB (431.4 KiB unpacked)
Retrieving: perl-IPC-Run-0.99-lp151.2.1.noarch.rpm [done]
Retrieving package python2-cffi-1.11.2-lp151.4.3.x86_64 (84/101), 230.7 KiB (  1.0 MiB unpacked)
Retrieving: python2-cffi-1.11.2-lp151.4.3.x86_64.rpm [done]
Retrieving package moreutils-0.63-lp151.1.1.x86_64 (85/101),  71.7 KiB (165.2 KiB unpacked)
Retrieving: moreutils-0.63-lp151.1.1.x86_64.rpm [.done (15.6 KiB/s)]
Retrieving package python2-xattr-0.9.2-lp151.2.3.x86_64 (86/101),  34.4 KiB (135.1 KiB unpacked)
Retrieving: python2-xattr-0.9.2-lp151.2.3.x86_64.rpm [done]
Retrieving package python-six-1.10.0-lp151.1.3.noarch (87/101),  27.1 KiB ( 62.0 KiB unpacked)
Retrieving: python-six-1.10.0-lp151.1.3.noarch.rpm [.done]
Retrieving package python-setuptools-20.2.2-lp151.2.3.noarch (88/101), 364.0 KiB (  1.4 MiB unpacked)
Retrieving: python-setuptools-20.2.2-lp151.2.3.noarch.rpm [done]
Retrieving package systemd-234-lp151.26.16.1.x86_64 (89/101),   2.6 MiB (  9.4 MiB unpacked)
Retrieving: systemd-234-lp151.26.16.1.x86_64.rpm [..done (2.6 MiB/s)]
Retrieving package glib2-tools-2.54.3-lp151.7.3.1.x86_64 (90/101), 133.3 KiB (285.9 KiB unpacked)
Retrieving: glib2-tools-2.54.3-lp151.7.3.1.x86_64.rpm [done]
Retrieving package go1.11-1.11.13-lp151.2.18.1.x86_64 (91/101),  98.3 MiB (385.4 MiB unpacked)
Retrieving: go1.11-1.11.13-lp151.2.18.1.x86_64.rpm [..........................................done (2.3 MiB/s)]
Retrieving package udev-234-lp151.26.16.1.x86_64 (92/101),   1.2 MiB (  7.4 MiB unpacked)
Retrieving: udev-234-lp151.26.16.1.x86_64.rpm [.done (2.4 KiB/s)]
Retrieving package libgio-2_0-0-2.54.3-lp151.7.3.1.x86_64 (93/101), 554.4 KiB (  1.6 MiB unpacked)
Retrieving: libgio-2_0-0-2.54.3-lp151.7.3.1.x86_64.rpm [.done]
Retrieving package go-1.12-lp151.2.3.1.x86_64 (94/101),  33.5 KiB (   82   B unpacked)
Retrieving: go-1.12-lp151.2.3.1.x86_64.rpm [done]
Retrieving package openssh-7.9p1-lp151.4.12.1.x86_64 (95/101), 990.1 KiB (  5.5 MiB unpacked)
Retrieving: openssh-7.9p1-lp151.4.12.1.x86_64.rpm [.done (2.0 MiB/s)]
Retrieving package glib-networking-2.54.1-lp151.3.3.1.x86_64 (96/101),  76.5 KiB (205.9 KiB unpacked)
Retrieving: glib-networking-2.54.1-lp151.3.3.1.x86_64.rpm [done]
Retrieving package git-core-2.26.2-lp151.4.12.1.x86_64 (97/101),   4.9 MiB ( 30.4 MiB unpacked)
Retrieving: git-core-2.26.2-lp151.4.12.1.x86_64.rpm [....done (1.5 MiB/s)]
Retrieving package git-2.26.2-lp151.4.12.1.x86_64 (98/101),  80.3 KiB (  3.2 KiB unpacked)
Retrieving: git-2.26.2-lp151.4.12.1.x86_64.rpm [done]
Retrieving package libsoup-2_4-1-2.62.2-lp151.3.1.x86_64 (99/101), 314.5 KiB (  1.1 MiB unpacked)
Retrieving: libsoup-2_4-1-2.62.2-lp151.3.1.x86_64.rpm [.done]
Retrieving package libostree-1-1-2018.9-lp151.1.1.x86_64 (100/101), 327.0 KiB (673.9 KiB unpacked)
Retrieving: libostree-1-1-2018.9-lp151.1.1.x86_64.rpm [.done (18.8 KiB/s)]
Retrieving package skopeo-1.0.0-lp151.1.1.x86_64 (101/101),   6.2 MiB ( 28.0 MiB unpacked)
Retrieving: skopeo-1.0.0-lp151.1.1.x86_64.rpm [......done (1.0 MiB/s)]

Checking for file conflicts: [..........done]
(  1/101) Installing: go-mtree-0.4.4-lp151.1.17.x86_64 [............done]
(  2/101) Installing: libcontainers-common-20200603-lp151.2.1.noarch [...........done]
(  3/101) Installing: oci-image-tools-0.3.0-lp151.1.17.x86_64 [............done]
(  4/101) Installing: oci-runtime-tools-0.2.0+r40-lp151.1.17.x86_64 [............done]
(  5/101) Installing: attr-2.4.47-lp151.3.70.x86_64 [..........done]
(  6/101) Installing: blog-2.18-lp151.5.3.x86_64 [........done]
(  7/101) Installing: gnu_parallel-20180422-lp151.2.1.noarch [...........done]
(  8/101) Installing: gsettings-desktop-schemas-3.24.1-lp151.3.2.x86_64 [...........done]
(  9/101) Installing: kbd-legacy-2.0.4-lp151.8.1.noarch [............done]
( 10/101) Installing: kmod-25-lp151.6.1.x86_64 [............done]
( 11/101) Installing: libargon2-1-0.0+git20171227.670229c-lp151.3.3.x86_64 [....done]
( 12/101) Installing: libcap-progs-2.25-lp151.3.70.x86_64 [.......done]
( 13/101) Installing: libedit0-3.1.snap20150325-lp151.3.3.x86_64 [..........done]
( 14/101) Installing: libgdbm4-1.12-lp151.2.67.x86_64 [...........done]
( 15/101) Installing: libisl15-0.18-lp151.2.70.x86_64 [............done]
( 16/101) Installing: libjson-c3-0.13-lp151.2.3.x86_64 [.....done]
( 17/101) Installing: libkmod2-25-lp151.6.1.x86_64 [.........done]
( 18/101) Installing: libmpfr6-4.0.1-lp151.2.70.x86_64 [...........done]
( 19/101) Installing: libmpx2-8.2.1+r264010-lp151.1.33.x86_64 [....done]
( 20/101) Installing: libmpxwrappers2-8.2.1+r264010-lp151.1.33.x86_64 [.....done]
( 21/101) Installing: libonig4-6.7.0-lp151.2.2.x86_64 [...........done]
( 22/101) Installing: libpcre2-8-0-10.31-lp151.2.3.x86_64 [..........done]
( 23/101) Installing: libqrencode4-4.0.0-lp151.2.3.x86_64 [.....done]
( 24/101) Installing: libsha1detectcoll1-1.0.3-lp151.3.3.x86_64 [......done]
( 25/101) Installing: linux-glibc-devel-4.15-lp151.2.72.noarch [............done]
( 26/101) Installing: make-4.2.1-lp151.7.39.x86_64 [..........done]
( 27/101) Installing: pam-config-0.96-lp151.1.2.x86_64 [...........done]
Additional rpm output:
*** write_config (account, /etc/pam.d/common-account-pc, ...)
**** write config for pam_access.so (account, disabled)
**** write config for pam_unix2.so (account, disabled)
**** write config for pam_unix.so (account, enabled)
**** write config for pam_krb5.so (account, disabled)
**** write config for pam_localuser.so (account, disabled)
**** write config for pam_sss.so (account, disabled)
**** write config for pam_ldap.so (account, disabled)
**** write config for pam_nam.so (account, disabled)
**** write config for pam_winbind.so (account, disabled)
**** write config for pam_time.so (account, disabled)
*** write_config (auth, /etc/pam.d/common-auth-pc, ...)
**** write config for pam_env.so (auth, enabled)
**** write config for pam_group.so (auth, disabled)
**** write config for pam_pkcs11.so (auth, disabled)
**** write config for pam_fp.so (auth, disabled)
**** write config for pam_fprint.so (auth, disabled)
**** write config for pam_fprintd.so (auth, disabled)
**** write config for pam_thinkfinger.so (auth, disabled)
**** write config for pam_gnome_keyring.so (auth, disabled)
**** write config for pam_kwallet5.so (auth, disabled)
**** write config for pam_ssh.so (auth, disabled)
**** write config for pam_unix2.so (auth, disabled)
**** write config for pam_unix.so (auth, enabled)
**** write config for pam_ecryptfs.so (auth, disabled)
**** write config for pam_krb5.so (auth, disabled)
**** write config for pam_sss.so (auth, disabled)
**** write config for pam_ldap.so (auth, disabled)
**** write config for pam_nam.so (auth, disabled)
**** write config for pam_winbind.so (auth, disabled)
*** write_config (password, /etc/pam.d/common-password-pc, ...)
**** write config for pam_winbind.so (password, disabled)
**** write config for pam_pwcheck.so (password, disabled)
**** write config for pam_passwdqc.so (password, disabled)
**** write config for pam_cracklib.so (password, enabled)
**** write config for pam_pwhistory.so (password, disabled)
**** write config for pam_gnome_keyring.so (password, disabled)
**** write config for pam_kwallet5.so (password, disabled)
**** write config for pam_ecryptfs.so (password, disabled)
**** write config for pam_unix2.so (password, disabled)
**** write config for pam_unix.so (password, enabled)
**** write config for pam_make.so (password, disabled)
**** write config for pam_exec.so (password, disabled)
**** write config for pam_krb5.so (password, disabled)
**** write config for pam_sss.so (password, disabled)
**** write config for pam_ldap.so (password, disabled)
**** write config for pam_nam.so (password, disabled)
*** write_config (session, /etc/pam.d/common-session-pc, ...)
**** write config for pam_selinux.so (session, disabled)
**** write config for pam_mkhomedir.so (session, disabled)
**** write config for pam_systemd.so (session, disabled)
**** write config for pam_limits.so (session, enabled)
**** write config for pam_unix2.so (session, disabled)
**** write config for pam_unix.so (session, enabled)
**** write config for pam_apparmor.so (session, disabled)
**** write config for pam_krb5.so (session, disabled)
**** write config for pam_sss.so (session, disabled)
**** write config for pam_ldap.so (session, disabled)
**** write config for pam_winbind.so (session, disabled)
**** write config for pam_nam.so (session, disabled)
**** write config for pam_umask.so (session, enabled)
**** write config for pam_ssh.so (session, disabled)
**** write config for pam_selinux.so (session, disabled)
**** write config for pam_gnome_keyring.so (session, disabled)
**** write config for pam_kwallet5.so (session, disabled)
**** write config for pam_exec.so (session, disabled)
**** write config for pam_ecryptfs.so (session, disabled)
**** write config for pam_env.so (session, enabled)


( 28/101) Installing: pkg-config-0.29.2-lp151.2.70.x86_64 [...........done]
( 29/101) Installing: systemd-presets-common-SUSE-15-lp151.7.1.noarch [......done]
( 30/101) Installing: tar-1.30-lp151.2.1.x86_64 [...........done]
( 31/101) Installing: sysvinit-tools-2.88+-lp151.2.3.x86_64 [...........done]
( 32/101) Installing: kbd-2.0.4-lp151.8.1.x86_64 [............done]
Additional rpm output:
Updating /etc/sysconfig/console ...
Updating /etc/sysconfig/keyboard ...


( 33/101) Installing: libmpc3-1.1.0-lp151.2.70.x86_64 [......done]
( 34/101) Installing: systemd-presets-branding-openSUSE-12.2-lp151.13.2.noarch [.....done]
( 35/101) Installing: binutils-2.32-lp151.3.6.1.x86_64 [............done]
Additional rpm output:
update-alternatives: using /usr/bin/ld.bfd to provide /usr/bin/ld (ld) in auto mode


( 36/101) Installing: file-5.32-lp151.8.3.1.x86_64 [.......done]
( 37/101) Installing: libapparmor1-2.12.3-lp151.4.3.1.x86_64 [.....done]
( 38/101) Installing: libarchive13-3.3.2-lp151.5.3.1.x86_64 [...........done]
( 39/101) Installing: libasan4-7.5.0+r278197-lp151.2.12.1.x86_64 [............done]
( 40/101) Installing: libatomic1-9.3.1+git1296-lp151.2.2.x86_64 [....done]
( 41/101) Installing: libcilkrts5-7.5.0+r278197-lp151.2.12.1.x86_64 [.......done]
( 42/101) Installing: libdb-4_8-4.8.30-lp151.7.3.1.x86_64 [............done]
( 43/101) Installing: libdbus-1-3-1.12.2-lp151.4.3.1.x86_64 [.........done]
( 44/101) Installing: libdevmapper1_03-1.02.149-lp151.4.12.1.x86_64 [.........done]
( 45/101) Installing: libexpat1-2.2.5-lp151.3.6.1.x86_64 [.........done]
( 46/101) Installing: libglib-2_0-0-2.54.3-lp151.7.3.1.x86_64 [............done]
( 47/101) Installing: libgomp1-9.3.1+git1296-lp151.2.2.x86_64 [...........done]
( 48/101) Installing: libitm1-9.3.1+git1296-lp151.2.2.x86_64 [.......done]
( 49/101) Installing: liblsan0-9.3.1+git1296-lp151.2.2.x86_64 [.........done]
( 50/101) Installing: libpython2_7-1_0-2.7.17-lp151.10.17.1.x86_64 [............done]
( 51/101) Installing: libpython3_6m1_0-3.6.10-lp151.6.18.1.x86_64 [............done]
( 52/101) Installing: libseccomp2-2.4.1-lp151.3.3.1.x86_64 [........done]
( 53/101) Installing: libtsan0-9.3.1+git1296-lp151.2.2.x86_64 [...........done]
( 54/101) Installing: libubsan0-7.5.0+r278197-lp151.2.12.1.x86_64 [.........done]
( 55/101) Installing: openslp-2.0.0-lp151.7.9.1.x86_64 [........done]
( 56/101) Installing: python-rpm-macros-20200207.5feb6c1-lp151.2.6.1.noarch [.....done]
( 57/101) Installing: libjq1-1.6-lp151.3.3.1.x86_64 [.........done]
( 58/101) Installing: glibc-devel-2.26-lp151.19.15.1.x86_64 [............done]
( 59/101) Installing: cpp7-7.5.0+r278197-lp151.2.12.1.x86_64 [............done]
( 60/101) Installing: perl-5.26.1-lp151.9.3.1.x86_64 [............done]
( 61/101) Installing: dbus-1-1.12.2-lp151.4.3.1.x86_64 [...........done]
Additional rpm output:
update-alternatives: using /usr/bin/dbus-launch.nox11 to provide /usr/bin/dbus-launch (dbus-launch) in auto mode


( 62/101) Installing: libgobject-2_0-0-2.54.3-lp151.7.3.1.x86_64 [.........done]
( 63/101) Installing: libgmodule-2_0-0-2.54.3-lp151.7.3.1.x86_64 [.....done]
( 64/101) Installing: rsync-3.1.3-lp151.4.3.1.x86_64 [............done]
( 65/101) Installing: python3-base-3.6.10-lp151.6.18.1.x86_64 [............done]
( 66/101) Installing: python-base-2.7.17-lp151.10.17.1.x86_64 [............done]
( 67/101) Installing: jq-1.6-lp151.3.3.1.x86_64 [......done]
( 68/101) Installing: gcc7-7.5.0+r278197-lp151.2.12.1.x86_64 [............done]
( 69/101) Installing: perl-TimeDate-2.30-lp151.3.6.1.noarch [...........done]
( 70/101) Installing: python3-3.6.10-lp151.6.18.1.x86_64 [........done]
( 71/101) Installing: python-xml-2.7.17-lp151.10.17.1.x86_64 [............done]
( 72/101) Installing: bats-1.1.0+cyphar-lp151.21.1.noarch [........done]
( 73/101) Installing: less-530-lp151.2.38.x86_64 [...........done]
( 74/101) Installing: libcryptsetup12-2.0.5-lp151.1.17.x86_64 [.........done]
( 75/101) Installing: shared-mime-info-1.9-lp151.3.20.x86_64 [............done]
( 76/101) Installing: cpp-7-lp151.3.5.x86_64 [......done]
( 77/101) Installing: perl-Time-Duration-1.200000-lp151.2.1.noarch [......done]
( 78/101) Installing: perl-IO-Tty-1.12-lp151.2.2.x86_64 [........done]
( 79/101) Installing: perl-Error-0.17025-lp151.2.1.noarch [........done]
( 80/101) Installing: python2-pycparser-2.17-lp151.2.1.noarch [............done]
( 81/101) Installing: gio-branding-openSUSE-42.1-lp151.4.2.noarch [......done]
( 82/101) Installing: gcc-7-lp151.3.5.x86_64 [...........done]
( 83/101) Installing: perl-IPC-Run-0.99-lp151.2.1.noarch [..........done]
( 84/101) Installing: python2-cffi-1.11.2-lp151.4.3.x86_64 [............done]
( 85/101) Installing: moreutils-0.63-lp151.1.1.x86_64 [...........done]
( 86/101) Installing: python2-xattr-0.9.2-lp151.2.3.x86_64 [...........done]
Additional rpm output:
update-alternatives: using /usr/bin/xattr-2.7 to provide /usr/bin/xattr (xattr) in auto mode


( 87/101) Installing: python-six-1.10.0-lp151.1.3.noarch [......done]
( 88/101) Installing: python-setuptools-20.2.2-lp151.2.3.noarch [............done]
Additional rpm output:
update-alternatives: using /usr/bin/easy_install-2.7 to provide /usr/bin/easy_install (easy_install) in auto mode


( 89/101) Installing: systemd-234-lp151.26.16.1.x86_64 [............done]
Additional rpm output:
Creating group systemd-journal with gid 486.
Creating group systemd-timesync with gid 485.
Creating user systemd-timesync (systemd Time Synchronization) with uid 485 and gid 485.
Creating group systemd-coredump with gid 484.
Creating user systemd-coredump (systemd Core Dumper) with uid 484 and gid 484.
Creating group systemd-network with gid 483.
Creating user systemd-network (systemd Network Management) with uid 483 and gid 483.
[/usr/lib/tmpfiles.d/journal-nocow.conf:26] Failed to resolve specifier: uninitialized /etc detected, skipping
All rules containing unresolvable specifiers will be skipped.
[/usr/lib/tmpfiles.d/tmp.conf:13] Duplicate line for path "/var/tmp", ignoring.
[/usr/lib/tmpfiles.d/var.conf:21] Duplicate line for path "/var/lib", ignoring.
[/usr/lib/tmpfiles.d/var.conf:23] Duplicate line for path "/var/spool", ignoring.
Created symlink /etc/systemd/system/multi-user.target.wants/remote-fs.target -> /usr/lib/systemd/system/remote-fs.target.
Created symlink /etc/systemd/system/getty.target.wants/getty@tty1.service -> /usr/lib/systemd/system/getty@.service.


( 90/101) Installing: glib2-tools-2.54.3-lp151.7.3.1.x86_64 [..........done]
( 91/101) Installing: go1.11-1.11.13-lp151.2.18.1.x86_64 [.............done]
Additional rpm output:
update-alternatives: using /usr/lib64/go/1.11/bin/go to provide /usr/bin/go (go) in auto mode


( 92/101) Installing: udev-234-lp151.26.16.1.x86_64 [............done]
( 93/101) Installing: libgio-2_0-0-2.54.3-lp151.7.3.1.x86_64 [............done]
( 94/101) Installing: go-1.12-lp151.2.3.1.x86_64 [.....done]
( 95/101) Installing: openssh-7.9p1-lp151.4.12.1.x86_64 [............done]
Additional rpm output:
Updating /etc/sysconfig/ssh ...
Failed to connect to bus: No such file or directory
setting /etc/ssh/sshd_config to root:root 0640. (wrong permissions 0600)


( 96/101) Installing: glib-networking-2.54.1-lp151.3.3.1.x86_64 [.........done]
( 97/101) Installing: git-core-2.26.2-lp151.4.12.1.x86_64 [............done]
( 98/101) Installing: git-2.26.2-lp151.4.12.1.x86_64 [.....done]
( 99/101) Installing: libsoup-2_4-1-2.62.2-lp151.3.1.x86_64 [............done]
(100/101) Installing: libostree-1-1-2018.9-lp151.1.1.x86_64 [..........done]
(101/101) Installing: skopeo-1.0.0-lp151.1.1.x86_64 [............done]
Executing %posttrans script 'blog-2.18-lp151.5.3.x86_64.rpm' [....
Output of systemd-presets-common-SUSE-15-lp151.7.1.noarch.rpm %posttrans script:
    Created symlink /etc/systemd/system/multi-user.target.wants/kbdsettings.service -> /usr/lib/systemd/system/kbdsettings.service.
    Created symlink /etc/systemd/system/default.target.wants/ca-certificates.path -> /usr/lib/systemd/system/ca-certificates.path.

.....
Output of udev-234-lp151.26.16.1.x86_64.rpm %posttrans script:
    Creating /lib/udev -> /usr/lib/udev symlink.

..done]
Removing intermediate container eb4ec335d7ae
 ---> 338bae9eee3b
Step 5/13 : ENV GOPATH /go
 ---> Running in ba30c408025a
Removing intermediate container ba30c408025a
 ---> cb87f9daaea0
Step 6/13 : ENV PATH $GOPATH/bin:$PATH
 ---> Running in c11fbe2ad96f
Removing intermediate container c11fbe2ad96f
 ---> 67a74c670c83
Step 7/13 : RUN go get -u github.com/cpuguy83/go-md2man &&     go get -u golang.org/x/lint/golint &&     go get -u github.com/securego/gosec/cmd/gosec
 ---> Running in 0906bce287c0
Removing intermediate container 0906bce287c0
 ---> acbe31045506
Step 8/13 : ENV SOURCE_IMAGE=/opensuse SOURCE_TAG=latest
 ---> Running in 1db4db3a3648
Removing intermediate container 1db4db3a3648
 ---> 5360420156c5
Step 9/13 : ARG DOCKER_IMAGE=opensuse/amd64:tumbleweed
 ---> Running in e5c6ca35128a
Removing intermediate container e5c6ca35128a
 ---> f7fc094bde1d
Step 10/13 : RUN skopeo copy docker://$DOCKER_IMAGE oci:$SOURCE_IMAGE:$SOURCE_TAG
 ---> Running in 19757956e3c0
Getting image source signatures
Copying blob sha256:5bc277bf9110c5f928249d6cfdc0f9beb075e9c03a0e9e5a0e6d4612ab6c3887
Copying config sha256:111b1c71d938c024da7d06779d0c3e714cb02f38867e7f4c464a4f48a39eb1ad
Writing manifest to image destination
Storing signatures
Removing intermediate container 19757956e3c0
 ---> 61bb66ce2db3
Step 11/13 : VOLUME ["/go/src/github.com/openSUSE/umoci"]
 ---> Running in c78b00a7d989
Removing intermediate container c78b00a7d989
 ---> eafb657e9d90
Step 12/13 : WORKDIR /go/src/github.com/openSUSE/umoci
 ---> Running in 0aa108a13721
Removing intermediate container 0aa108a13721
 ---> 2a07d6118410
Step 13/13 : COPY . /go/src/github.com/openSUSE/umoci
 ---> 25269e6329e7
Successfully built 25269e6329e7
Successfully tagged umoci_dev:latest
docker run --rm -it --security-opt apparmor:unconfined --security-opt label:disable -v :/go/src/github.com/openSUSE/umoci umoci_dev make local-doc
go-md2man -in doc/man/umoci.1.md -out doc/man/umoci.1
go-md2man -in doc/man/umoci-unpack.1.md -out doc/man/umoci-unpack.1
go-md2man -in doc/man/umoci-tag.1.md -out doc/man/umoci-tag.1
go-md2man -in doc/man/umoci-stat.1.md -out doc/man/umoci-stat.1
go-md2man -in doc/man/umoci-rm.1.md -out doc/man/umoci-rm.1
go-md2man -in doc/man/umoci-repack.1.md -out doc/man/umoci-repack.1
go-md2man -in doc/man/umoci-remove.1.md -out doc/man/umoci-remove.1
go-md2man -in doc/man/umoci-raw.1.md -out doc/man/umoci-raw.1
go-md2man -in doc/man/umoci-raw-unpack.1.md -out doc/man/umoci-raw-unpack.1
go-md2man -in doc/man/umoci-raw-runtime-config.1.md -out doc/man/umoci-raw-runtime-config.1
go-md2man -in doc/man/umoci-raw-config.1.md -out doc/man/umoci-raw-config.1
go-md2man -in doc/man/umoci-raw-add-layer.1.md -out doc/man/umoci-raw-add-layer.1
go-md2man -in doc/man/umoci-new.1.md -out doc/man/umoci-new.1
go-md2man -in doc/man/umoci-ls.1.md -out doc/man/umoci-ls.1
go-md2man -in doc/man/umoci-list.1.md -out doc/man/umoci-list.1
go-md2man -in doc/man/umoci-insert.1.md -out doc/man/umoci-insert.1
go-md2man -in doc/man/umoci-init.1.md -out doc/man/umoci-init.1
go-md2man -in doc/man/umoci-gc.1.md -out doc/man/umoci-gc.1
go-md2man -in doc/man/umoci-config.1.md -out doc/man/umoci-config.1
install -D -m0755 umoci //usr/bin/umoci
for man in doc/man/umoci-unpack.1 doc/man/umoci-remove.1 doc/man/umoci-config.1 doc/man/umoci-ls.1 doc/man/umoci-raw-add-layer.1 doc/man/umoci-repack.1 doc/man/umoci-new.1 doc/man/umoci-gc.1 doc/man/umoci-raw-config.1 doc/man/umoci-tag.1 doc/man/umoci-rm.1 doc/man/umoci-init.1 doc/man/umoci-raw.1 doc/man/umoci-stat.1 doc/man/umoci-raw-unpack.1 doc/man/umoci-raw-runtime-config.1 doc/man/umoci-insert.1 doc/man/umoci.1 doc/man/umoci-list.1; do \
	filename="$(basename -- "$man")"; \
	target="//usr/share/man/man${filename##*.}/$filename"; \
	install -D -m0644 "$man" "$target"; \
	gzip -9f "$target"; \
 done
install: cannot stat 'doc/man/umoci-unpack.1': No such file or directory
gzip: //usr/share/man/man1/umoci-unpack.1: No such file or directory
install: cannot stat 'doc/man/umoci-remove.1': No such file or directory
gzip: //usr/share/man/man1/umoci-remove.1: No such file or directory
install: cannot stat 'doc/man/umoci-config.1': No such file or directory
gzip: //usr/share/man/man1/umoci-config.1: No such file or directory
install: cannot stat 'doc/man/umoci-ls.1': No such file or directory
gzip: //usr/share/man/man1/umoci-ls.1: No such file or directory
install: cannot stat 'doc/man/umoci-raw-add-layer.1': No such file or directory
gzip: //usr/share/man/man1/umoci-raw-add-layer.1: No such file or directory
install: cannot stat 'doc/man/umoci-repack.1': No such file or directory
gzip: //usr/share/man/man1/umoci-repack.1: No such file or directory
install: cannot stat 'doc/man/umoci-new.1': No such file or directory
gzip: //usr/share/man/man1/umoci-new.1: No such file or directory
install: cannot stat 'doc/man/umoci-gc.1': No such file or directory
gzip: //usr/share/man/man1/umoci-gc.1: No such file or directory
install: cannot stat 'doc/man/umoci-raw-config.1': No such file or directory
gzip: //usr/share/man/man1/umoci-raw-config.1: No such file or directory
install: cannot stat 'doc/man/umoci-tag.1': No such file or directory
gzip: //usr/share/man/man1/umoci-tag.1: No such file or directory
install: cannot stat 'doc/man/umoci-rm.1': No such file or directory
gzip: //usr/share/man/man1/umoci-rm.1: No such file or directory
install: cannot stat 'doc/man/umoci-init.1': No such file or directory
gzip: //usr/share/man/man1/umoci-init.1: No such file or directory
install: cannot stat 'doc/man/umoci-raw.1': No such file or directory
gzip: //usr/share/man/man1/umoci-raw.1: No such file or directory
install: cannot stat 'doc/man/umoci-stat.1': No such file or directory
gzip: //usr/share/man/man1/umoci-stat.1: No such file or directory
install: cannot stat 'doc/man/umoci-raw-unpack.1': No such file or directory
gzip: //usr/share/man/man1/umoci-raw-unpack.1: No such file or directory
install: cannot stat 'doc/man/umoci-raw-runtime-config.1': No such file or directory
gzip: //usr/share/man/man1/umoci-raw-runtime-config.1: No such file or directory
install: cannot stat 'doc/man/umoci-insert.1': No such file or directory
gzip: //usr/share/man/man1/umoci-insert.1: No such file or directory
install: cannot stat 'doc/man/umoci.1': No such file or directory
gzip: //usr/share/man/man1/umoci.1: No such file or directory
install: cannot stat 'doc/man/umoci-list.1': No such file or directory
gzip: //usr/share/man/man1/umoci-list.1: No such file or directory
Makefile:80: recipe for target 'install' failed
make: [install] Error 1 (ignored)

```

--- a/.travis.yml
+++ b/.travis.yml
@@ -10,6 +10,7 @@ go:
 
 before_install:
   - sudo sh -c "apt-get -qq update && apt-get install -y gcc-multilib"
+  - go get -u -v github.com/cpuguy83/go-md2man
 
 env:
   - DOCKER_IMAGE="opensuse/leap:latest"

--- a/Makefile
+++ b/Makefile
@@ -76,7 +76,7 @@ release:
 	hack/release.sh -v $(VERSION) -S "$(GPG_KEYID)"
 
 .PHONY: install
-install: umoci doc
+install: umoci docs
 	install -D -m0755 umoci $(DESTDIR)/$(BINDIR)/umoci
 	-for man in $(MANPAGES); do \
 		filename="$$(basename -- "$$man")"; \
@@ -134,18 +134,14 @@ local-validate-build:
 	env CGO_ENABLED=0 $(GO) build ${STATIC_BUILD_FLAGS} -o /dev/null ${CMD}
 	$(GO) test -run nothing ${DYN_BUILD_FLAGS} $(PROJECT)/...
 
-.PHONY: doc
-doc: umociimage
-	$(DOCKER_RUN) $(UMOCI_IMAGE) make local-doc
-
 MANPAGES_MD := $(wildcard doc/man/*.md)
 MANPAGES    := $(MANPAGES_MD:%.md=%)
 
 doc/man/%.1: doc/man/%.1.md
 	$(GO_MD2MAN) -in $< -out $@
 
-.PHONY: local-doc
-local-doc: $(MANPAGES)
+.PHONY: docs
+docs: $(MANPAGES)
 
 # Used for tests.
 DOCKER_IMAGE :=opensuse/amd64:tumbleweed
@@ -171,7 +167,7 @@ local-test-unit:
 .PHONY: test-integration
 test-integration: umociimage
 	touch $(COVERAGE) && chmod a+rw $(COVERAGE)
-	$(DOCKER_RUN) -e COVERAGE=$(COVERAGE) $(UMOCI_IMAGE) make TESTS="${TESTS}" local-test-integration
+	$(DOCKER_RUN) -e COVERAGE=$(COVERAGE)                             $(UMOCI_IMAGE) make TESTS="${TESTS}" local-test-integration
 	$(DOCKER_RUN) -e COVERAGE=$(COVERAGE) -u 1000:1000 --cap-drop=all $(UMOCI_IMAGE) make TESTS="${TESTS}" local-test-integration
 
 .PHONY: local-test-integration
@@ -182,5 +178,5 @@ shell: umociimage
 	$(DOCKER_RUN) $(UMOCI_IMAGE) bash
 
 .PHONY: ci
-ci: umoci umoci.cover doc validate test-unit test-integration
+ci: umoci umoci.cover docs validate test-unit test-integration
 	hack/ci-coverage.sh $(COVERAGE)

